common/api/core-react.api.md

```diff
@@ -411,6 +412,10 @@ export class Dialog extends React_2.Component<DialogProps, DialogState> {
     // (undocumented)
     static defaultProps: Partial<DialogProps>;
     // (undocumented)
+    protected getFooterButtons(buttonCluster: DialogButtonDef[] | undefined, primaryStyleType?: ButtonProps["styleType"], noCoreButtonClasses?: boolean): React_2.ReactNode[] | undefined;
```

Comment:
Could we not add these as public APIs? I.e. add @internal OR export this as an @internal DialogFooterButtons component.